### PR TITLE
Adds 'Subscribe to our newsletter' section, before footer.

### PR DESCRIPTION
**Pull request by joaojusto**
screen shot 2015-10-19 at 17 27 09

### CODE REVIEW

**Comment by naps62** [comment]
printscreen please?

**Comment by joaojusto** [comment]
@naps62 updated, sorry you had to ask :disappointed:

File: source/partials/_newsletter.slim [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    .GridCell-content
+      .ContentFormatter.ContentFormatter--centerHorizontally.u-smallMargin
+        .Text.Text--strong.Text--dark.u-xSmallMargin
+          | Subscribe to our weekly newsletter
+        .Text
+          | Where we share articles, videos or websites that got us excited over the week.
+
+form.Newsletter target='_blank' action="//subvisual.us5.list-manage.com/subscribe/post?u=79f7816bac08905f81c1a8689&amp;id=f8b8f31221" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form"
+  .Newsletter-inputWrapper
+    input.Newsletter-input type='email' value='' placeholder='Your email address' id="mce-EMAIL" name="EMAIL"
```
Comment (zamith) [inline source/partials/_newsletter.slim]: This input is missing a label, even if it is visually hidden, wouldn't you say?

Comment (joaojusto) [inline source/partials/_newsletter.slim]: Yes you'r right, I'll add it

**Comment by joaojusto** [comment]
@zamith bump

**Comment by joaojusto** [comment]
@zamith updated, I didn't knew that that component already existed.

**Comment by joaojusto** [comment]
Yes it can, I already changed it.

**Comment by zamith** [comment]
:+1:

File: source/stylesheets/components/_newsletter.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  .Newsletter-submit {
+    width: 100%;
+    padding: $Theme-spacing-small;
+  }
+}
+
+.Newsletter {
+  ::placeholder {
+    color: $Theme-color-poloBlue;
+    opacity: 1;
```
Comment (zamith) [inline source/stylesheets/components/_newsletter.scss]: I'm guessing this is a design decision, but why do we want a placeholder to be opaque? Doesn't it look like the input is already filled?

Comment (joaojusto) [inline source/stylesheets/components/_newsletter.scss]: Not necessarily but I guess you can question @fcbaila directly about this :stuck_out_tongue:

**Pull request merged**